client/src/views/Environment.tsx

```diff
@@ -71,7 +84,20 @@ function Environment() {
             >
               Open in Okteto
             </Button>
-          }
+          )}
+
+          <Button
+            sx={{
+              color: (theme) =>
+                theme.palette.mode === 'dark' ? '#fff' : '#000',
+            }}
+            variant="contained"
+            color="warning"
```

Comment:
Can we use the green color for the button here as it is defined in the mock? You can get the green value from the theme palette, like we do here: https://github.com/okteto/docker-desktop-extension/blob/3bccd6e720a468f535baa8e66d88cf6cd4459046/client/src/components/Link.tsx#L29

Reply:
Sure. Fixed.